src/Test/Html/Internal/ElmHtml/Query.elm

```diff
@@ -241,22 +239,12 @@ hasBoolAttribute attribute value facts =
 
 hasClass : String -> Facts msg -> Bool
 hasClass queryString facts =
-    List.member queryString (classnames facts)
-
-
-hasClassNS : String -> Facts msg -> Bool
-hasClassNS queryString facts =
-    List.member queryString (classnamesNS facts)
+    List.member queryString (classnames facts ++ classes facts)
```

Comment:
I both love this, and have questions.

- I love how clever and cleaner it is to my implementation. Awesome job!
Q1: We are touching how `hasClass` works and this might break peoples tests in weird and wonderful ways. Might using a more verbose abstraction that leaves current code paths intact be safer?

 Q2: `hasClassName` will mysteriously not work for svgs but work for html. Might that cause me confusion and make it difficult to reason about my test?

Reply:
> this might break peoples tests

Hm, I don't think this could break any existing tests, could it? 🤔 

It's doing a `List.member` on a strictly larger list, so is there any way it could fail when previously it succeeded?

Reply:
Also, what is ` hasExactClassName`? 😅

Reply:
```
[ test "does not find class on svg elements" <|
            \() ->
                let
                    svgClass =
                        "some-NS-class"
                in
                Svg.svg
                    [ SvgAttribs.class svgClass ]
                    [ Svg.circle [ SvgAttribs.cx "50", SvgAttribs.cy "50", SvgAttribs.r "40" ] [] ]
                    |> Query.fromHtml
                    |> Query.hasNot [ class svgClass ]
```
This is a test I wrote to assert that the svg class was not being found. With these changes that would break.

Despite how useful we think this test is, the fact is, the changes would break previously passing tests.

> Also, what is ` hasExactClassName`? 😅
hasExactClassName is an existing function that does not split the class names. So it will assert the entire class string. Here is NS equivalent I wrote to mimic the behaviour.
```
test "exactClassNameNS selector finds the exact class value on svg" <|
            \() ->
                let
                    svgClass =
                        "some-NS-class another-NS-class"
                in
                Svg.svg
                    [ SvgAttribs.class svgClass ]
                    [ Svg.circle [ SvgAttribs.cx "50", SvgAttribs.cy "50", SvgAttribs.r "40" ] [] ]
                    |> Query.fromHtml
                    |> Query.has [ exactClassNameNS svgClass ]
```

Reply:
Im certainly not going to make assumptions about how people test their code, just raising the point that we are touching a path that changes test behaviour on nodes other than svg, which was the focus of the fix. Whether this has other weird and wonderful side effects, not sure?

It seems you are favouring this approach? I am happy to support yours and @tesk9 's intuition on this.

@tesk9 Would you be keen to implement the `exactClassName` implementation?

Reply:
If we're sure we want `exactClassName` to match both attributes and properties, I'm happy to implement!